modules/geometry/line.hpp

```diff
@@ -177,6 +177,24 @@ inline T length(const Line &line) {
   return bg::length<T>(line.obj_);
 }
 
+inline Line rotate(const Line &line, float hdg) {
+  namespace trans = boost::geometry::strategy::transform;
+  trans::rotate_transformer<boost::geometry::radian, double, 2, 2> rotate(hdg);
+  Line line_rotated;
+  boost::geometry::transform(line.obj_, line_rotated.obj_, rotate);
+  line_rotated.recompute_s();
```

Comment:
The length(s) should not change due to a rotation?

Reply:
I know, just wanted to make sure ... I'll remove it